### DISCUSSION TITLE
Allowing faster completion of services

### DISCUSSION THREAD

**Discussion by Photon-GitHub** [tosdr/edit.tosdr.org#1017] [Ideas]
The current review section only shows the points which have been created most recently. This, however, makes the completion of old services with pending points basically impossible.
Therefore I suggest that some older points, preferrably ones of services that already have lots of other points approved, are injected into the list of most recent points so we can achieve both fast processing of new points as well as the ability to finish up some already extensively reviewed services.

**Comment by madoleary** [comment]
Would this not be solved by having a table of all the pending points, searchable by service? That way you can just search for the service that you want to review.

**Comment by Vinnl** [comment]
@madoleary We have that (though not searchable by service, I think): https://edit.tosdr.org/points/pending

(_Edit:_ Ah, even [by service is possible](https://edit.tosdr.org/services/803?scope=pending).)

The problem is: curators don't look for older points to review. So @Photon-GitHub has created a few points in the past that now never get reviewed.

(In fact, I think the main problem is that points are getting created quicker than we can review them. Otherwise, over time the "Points for you to review" list should be empty at some point in time. Perhaps we should focus more on reviewing than on fixing points from the import script? And perhaps attract additional curators, if there are suitable ones available still.)

**Comment by madoleary** [comment]
So...I made that table, haha, so I think it says something that I actually forgot about it, given I'm mostly working from the newer setup. Hence why I think the newer setup needs to change, which is perhaps a different discussion.

Is the problem "older points," or is the problem that we have services that are almost complete, and what's preventing them from completion is the fact that their outstanding pending points are deep in the backlog? In either case, we'd need to define "older" points, and also services that are "nearing completion."

I personally am still working through my changes-requested points. It takes some time, as we're all aware. I could switch gears to pending, but if the backlog is so huge, maybe it would be better to find a way to address some of these questions first.

Edit: @Vinnl agree about attracting curators.
